docs/EVE-IMAGE-REPRODUCIBILITY.md

```diff
@@ -0,0 +1,274 @@
+# EVE Reproducibility
+
+A key element of EVE is its consistency. Among other things, that means that it must be possible to rebuild EVE images, and the packages that are composed to form EVE, and get functionally identical bits.
+
+We write "functionally identical" rather than "precisely identical" or just "identical", because the compilation and composition process can change
+elements like timestamps or randomly generated data, leading to minor differences. Ideally, two builds of EVE from the same source would be
+precisely identical; that is a future goal. However, functionally identical images are a requirement for EVE.
+
+This document describes the reproducibility structures and process of EVE. It answers the "source-to-binary" question: 
+
+    given that I have a specific version of this source code repository, how does the process ensure that I will get an
```

Comment:
I think the statement is really "either produce a functionally eqivalent or fail".
For example, a curl/wget of a particular hashed piece of code might fail to find that if I try rebuilding the same eve tag a year from now. Don't know how likely it is that other parts disappear; the alpine caching will handle alpine pkg versions which are removed from alpine.

Don't know if you want to mention this "or fail" in this document.

Reply:
I actually do, that is a very good point. Getting it now.

Reply:
Done